cpg-language-go/src/main/kotlin/de/fraunhofer/aisec/cpg/passes/GoEvaluationOrderGraphPass.kt

```diff
@@ -114,7 +114,8 @@ class GoEvaluationOrderGraphPass(ctx: TranslationContext) : EvaluationOrderGraph
                 // It is a bit philosophical whether the deferred call happens before or after the
                 // return statement in the EOG. For now, it is easier to have it as the last node
                 // AFTER the return statement
-                addEOGEdge(path.last(), defer.input)
+                // Todo Propagate AssumptionsEdge
```

Comment:
I as the author of this pass have no idea what to do here

Reply:
This is a note to me, to not forget the propagation of assumptions. I will remove all todos of this type and note this places down on my machine.